### PR TITLE
Added a DAG to index geomedian data.

### PR DESCRIPTION
**Pull request by tebadi**

### CODE REVIEW

File: dags/s2_geomedian_indexing.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+with DAG(
+    "Sentinel-2-geomedian-backlog-indexing",
+    default_args=DEFAULT_ARGS,
+    schedule_interval=DEFAULT_ARGS["schedule_interval"],
+    tags=["Sentinel-2", "geomedian-indexing"],
+    catchup=False,
+) as dag:
+
+    # This needs to be updated in the future in case more zones have been added
+    x = range(153, 246)
```
Comment (Kirill888) [inline dags/s2_geomedian_indexing.py:76]: I'm a bit concerned about this, how will this be updated when content of db changes?

File: dags/s2_geomedian_indexing.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        INDEXING = KubernetesPodOperator(
+            namespace="processing",
+            image=INDEXER_IMAGE,
+            image_pull_policy="Always",
+            arguments=[
+                "s3-to-dc",
+                "--stac",
+                "--no-sign-request",
+                f"s3://deafrica-services/gm_s2_annual/1-0-0/x{index}/**/*.json",
+                "s2_l2a",
```
Comment (Kirill888) [inline dags/s2_geomedian_indexing.py:87]: `s2_l2a` is product name? it's not right

File: dags/s2_geomedian_indexing.py (hunk 1 above)
Comment (tebadi) [reply dags/s2_geomedian_indexing.py:76]: It won't.  It's very specific to this case.  May be I need to allow the user to specify this range as input parameters to the DAG.

File: dags/s2_geomedian_indexing.py (hunk 2 above)
Comment (tebadi) [reply dags/s2_geomedian_indexing.py:87]: I changed this, I might not have pushed it.

File: dags/s2_geomedian_indexing.py (hunk 1 above)
Comment (tebadi) [reply dags/s2_geomedian_indexing.py:76]: Will this address your concern?

**Review by Kirill888 — APPROVED** [review]

File: dags/s2_geomedian_indexing.py (hunk 1 above)
Comment (Kirill888) [reply dags/s2_geomedian_indexing.py:76]: this needs to be 246+1, range does not include last value and bucket has x246 in it right now

**Pull request merged**